File: FileHelpers.CodeExamples/Example CSharp/CustomerDelimited.cs
using System;
using FileHelpers;

namespace Examples
{

	[DelimitedRecord(",")]
	public class Customer
	{
		public int CustId;
		
		public string Name;

		public decimal Balance;

		[FieldConverter(ConverterKind.Date, "ddMMyyyy")]
		public DateTime AddedDate;

		#region "  only for the transformation example  "
		 
		[TransformToRecord(typeof(Customer2))]
		public Customer2 CreateSimilar()
		{
			Customer2 res = new Customer2();

			res.CustId = this.CustId;
			res.Name = this.Name;
			res.Balance = this.Balance;
			res.AddedDate = this.AddedDate;

			return res;
		}

		#endregion
	}

}

File: FileHelpers.CodeExamples/Example VbNet - VS 2005/CustomerDelimited.vb

'-> Never forget:  Imports FileHelpers
Imports FileHelpers

<DelimitedRecord(",")> _
Public Class Customer

    Public CustId As Integer
    Public Name As String
    Public Balance As Decimal
    <FieldConverter(ConverterKind.Date, "ddMMyyyy")> _
    Public AddedDate As DateTime



    '-> este método es para cambiar el formato no
    '-> no para el uso básico

    <TransformToRecord(GetType(Customer2))> _
      Public Function CrearSimilar() As Customer2
        Dim res As Customer2 = New Customer2

        '-> Simplemente copiamos los valores que nos interesan
        res.CustId = Me.CustId
        res.Name = Me.Name
        res.Balance = Me.Balance
        res.AddedDate = Me.AddedDate

        Return res
    End Function


End Class


